stories/topics/ref-gcp-decide-offer-type.adoc

```diff
@@ -0,0 +1,16 @@
+[id="ref-gcp-decide-offer-type"]
+
+= Deciding the offer type
+
+The following table lists the offer types and the corresponding GCP instance types. 
+Depending on the workload needs, you can choose a more suitable offer type for the extension nodes.
+
+[options="header",width="60%",cols="^s,<"]
+|====
+| Offer Type (nodes) | GCP Instance Type
+| 100 | n2-standard-2
+| 200 | n2-standard-4
+| 400 | n2-standard-8
```

Comment:
Can you left-justify the 1st column? This is how it renders in the build:
![image](https://github.com/ansible/aap-docs/assets/44700011/de65cb9a-c831-439e-a418-bd8ea0e90451)